src/framework/ui/toggle/toggle.component.tsx

```diff
@@ -61,10 +61,6 @@ export type ToggleElement = React.ReactElement<ToggleProps>;
  * Can be `primary`, `success`, `info`, `warning`, `danger`, `basic` or `control`.
```

Comment:
So now there are no any animations in the toggle?

Reply:
No color animations. Because it doesn't look good for `control` status. I think we need to reimplement this animation because using color animation "from border to background" doesn't look good for bright colors on dark backgrounds.